projects/storefrontlib/cms-components/myaccount/address-book/address-form/suggested-addresses-dialog/suggested-addresses-dialog.component.html

```diff
@@ -43,7 +43,9 @@
             [for]="'suggested-addresses--suggested-' + i"
           >
             {{ 'addressSuggestion.suggestedAddress' | cxTranslate }}
-            {{ suggestedAddresses?.length > 1 ? i + 1 : null }}
+            {{
+              suggestedAddresses && suggestedAddresses.length > 1 ? i + 1 : null
```

Comment:
What is the result when `suggestedAddresses` is falsy?

Should we use a default value instead? like 
```
(suggestedAddresses?.length ?? 0) > 1
```

Reply:
Also, I'm not sure about the reult with operator precedence.  The `>` is executed before the `&&` I think.

Reply:
I revert it back, `suggestedAddresses` is an array.